impl/src/main/java/org/ehcache/impl/internal/store/disk/OffHeapDiskStore.java

```diff
@@ -466,6 +463,7 @@ public void initAuthoritativeTier(AuthoritativeTier<?, ?> resource) {
    * This is kind of a hack, but it's safe to use this if the regular portability
    * is stateless.
    */
+  @SuppressWarnings("unchecked")
```

Comment:
But that makes me think I've never advocate my `Util.uncheckedCast` to resolve these issues without a broad `@SuppressWarnings` method annotation. :-)